src/components/settings-menu/BackupSection/AlreadyBackedUpView.js

```diff
@@ -238,7 +249,10 @@ export default function AlreadyBackedUpView() {
               size="large"
               weight="semibold"
             >
-              􀙶 Back up to {cloudPlatform}
+              􀙶{' '}
```

Comment:
confirming this is on purpose

Reply:
Correct, there is an SF Symbols icon of a cloud at the beginning of that line (which GitHub doesn't seem to render) so the space comes after that icon.